core/src/navigation/GlobalSearch.html

```diff
@@ -272,6 +293,11 @@
       inputElem,
       luigiCustomSearchRenderer__slot
     });
+
+    if (GenericHelpers.isFunction(search.searchProvider.toggleSearch)) {
+      let fieldVisible = !!isSearchFieldVisible ? !isSearchFieldVisible : true;
```

Comment:
1. What is `!!isSearchFieldVisible`?  
2. use `const fieldVisible` is more correct

Reply:
fieldVisible it is just a boolean: true if Input text searchbox is visible; false if it is not... 
I'll pass this value as parameter of the toggleSearch call back.
Yes, better change let to const.... hopefully with eslint it will do it for me....

Reply:
I meant why did you use double negative syntax? `!!isSearchFieldVisible`  It doesn't make sense.

Reply:
Default value of isSearchFieldVisible is undefined.... it means that it is the first time you click on search button, isSearchFieldVisible=undefined and I need to pass fieldVisible=true to callback function.

Reply:
OK. so why not write it as what you described, `(isSearchFieldVisible === undefined)`, which will be more understandable/readable

Reply:
Done